dusty/cli/repos.py

```diff
@@ -9,9 +9,9 @@
 repo. To override a repo, use the `override` or `from` commands.
 
 Usage:
-  repos from <source_path>
+  repos from [--all] <source_path>
   repos list
-  repos manage <repo_name>
+  repos manage (--all | <repo_name>)
   repos override <repo_name> <source_path>
   repos update
 
```

Comment:
add an Option:  that describes what --all does.  I do not know from what is above